config/environments/development.rb

```diff
@@ -18,10 +18,13 @@
   # Run rails dev:cache to toggle caching.
   if Rails.root.join('tmp', 'caching-dev.txt').exist?
     config.action_controller.perform_caching = true
-    config.cache_store = :memory_store
+
     config.public_file_server.headers = {
       'Cache-Control' => "public, max-age=#{2.days.to_i}"
     }
+
+    config.cache_store = :file_store
```

Comment:
Enabling `:file_store` in development because it is much easier to work with - you can see exactly what is cached on disk.